src/shared/services/auth/auth.test.js

```diff
@@ -278,4 +294,101 @@ describe('The auth service', () => {
       });
     });
   });
+
+  describe('Retrieving the auth domain', () => {
+    it('returns the auth domain', () => {
+      savedOauthDomain = 'datapunt';
+      const oauthDomain = getOauthDomain();
+
+      expect(oauthDomain).toEqual('datapunt');
+    });
+  });
+
+  describe('isAuthenticated', () => {
+    it('checks for logged in status', () => {
+      savedAccessToken = '123AccessToken';
+
+      expect(isAuthenticated()).toEqual(true);
+    });
+    it('checks for logged out status', () => {
+      savedAccessToken = '';
+
+      expect(isAuthenticated()).toEqual(false);
+    });
+  });
+
+  describe('getScopes', () => {
+    it('should return a an empty array', () => {
+      parseAccessToken.mockImplementation(() => ({}));
+
+      savedAccessToken = '123AccessToken';
+      initAuth();
+      const scopes = getScopes();
+
+      expect(scopes).toEqual([]);
+    });
+
+    it('should return the scopes', () => {
+      parseAccessToken.mockImplementation(() => ({
+        scopes: ['SIG/ALL']
+      }));
+
+      savedAccessToken = '123AccessToken';
+      initAuth();
+      const scopes = getScopes();
+
+      expect(scopes).toEqual(['SIG/ALL']);
+    });
+  });
+
+  describe('getName', () => {
+    it('should return a an empty string', () => {
+      parseAccessToken.mockImplementation(() => ({}));
+
+      savedAccessToken = '123AccessToken';
+      initAuth();
+      const name = getName();
+
+      expect(name).toEqual('');
+    });
+
+    it('should return the scopes', () => {
+      parseAccessToken.mockImplementation(() => ({
+        name: 'Jezus Christus'
```

Comment:
Maybe not the best test name for a public repository

Reply:
gedaan